packages/axelar-local-dev/src/Network.ts

```diff
@@ -203,6 +225,54 @@ export class Network {
         return this.create3Deployer;
     }
 
+    async deployInterchainTokenService() {
+        logger.log(`Deploying the InterchainTokenService for ${this.name}... `);
+        const deploymentSalt = keccak256('0x1234');
+        const wallet = this.ownerWallet;
+        const tokenManagerDeployer = await deployContract(wallet, TokenManagerDeployer, [this.create3Deployer.address]);
+        const standardizedTokenLockUnlock = await deployContract(wallet, StandardizedTokenLockUnlock);
+        const standardizedTokenMintBurn = await deployContract(wallet, StandardizedTokenMintBurn);
```

Comment:
can you use `Promise.all` to some of contract deployments here (which don't depends on other deployment) to make it a bit more efficient?